### PR TITLE
Ports my old pr, bee burst

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
it just fits here, right? bee?

original pr: https://github.com/Monkestation/MonkeStation/pull/532

Adds a drink, made with 3 bees knees and a sugar rush, that makes you start puking bees due to the pure honey in it.

## Why It's Good For The Game
Bees are fun to spawn, and honey/bees knees is rare enough that it would take a while to make some
Also its just hilarious

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

beedr_floor

beedr_ono

beedr_spill

</details>

## Changelog
:cl:
add: bee burst drink
/:cl:

### CODE REVIEW

**Comment by EvilDragonfiend** [comment]
that sounds difficult to call a port, but okay..?

and `chem_flags` is missing.

**Comment by etherware-novice** [comment]
oops

**Comment by FriendlyContractor** [comment]
- Testing photographs were clearly made on monkestation. You must test your code here
- Needlessly spawning mobs slows server down
- Only uselful for griefing

**Comment by etherware-novice** [comment]
griefing?

**Comment by BarteG44** [comment]
imo we do need a real reason to call ourselves beestation. even goon has more focus on bees than us

**Comment by DerpiestBruh** [comment]
beesed. speedmerge

**Comment by EvilDragonfiend** [comment]
It isn't fun type only. Check how other drinks are.

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 1]
```diff
@@ -959,3 +959,36 @@
 	M.adjust_disgust(30)
 	SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "quality_drink", /datum/mood_event/quality_bad)
 	. = ..()
+
+/datum/reagent/consumable/ethanol/bee_burst
+	name = "Bee Burst"
+	glass_name = "Bee Burst"
+	description = "A \"drink\" consisting of excess of sugar and honey. It's barely drinkable."
+	glass_desc = "There's so much honey in here that it would attract real bees. Drink at caution."
```
Comment (Penwin0) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: This description is not proper gramatically

**Review by Penwin0 — COMMENTED** [review]
Could you please update testing evidence with images taken on bee? that's clearly monkestation

**Pull request closed without merging**